augly/utils/expected_output/video_tests/expected_metadata.json

```diff
@@ -52,6 +52,62 @@
             "src_width": 1920
         }
     ],
+    "augment_audio": [
+        {
+            "name":"augment_audio",
+            "src_duration":10.027855,
+            "dst_duration":10.027855,
+            "src_fps":29.916666666666668,
+            "dst_fps":29.916666666666668,
+            "src_width":1920,
+            "src_height":1080,
+            "dst_width":1920,
+            "dst_height":1080,
+            "src_segments":[
+                {
+                    "start":0.0,
+                    "end":10.027855
+                }
+            ],
+            "dst_segments":[
+                    {
+                    "start":0.0,
+                    "end":10.027855
+                }
+            ],
+            "audio_metadata":[
+                {
+                    "name":"apply_lambda",
+                    "src_duration":2.82675,
+                    "dst_duration":2.82675,
+                    "src_num_channels":1,
+                    "dst_num_channels":1,
+                    "src_sample_rate":48000,
+                    "dst_sample_rate":48000,
+                    "src_segments":[
+                    {
+                        "start":0.0,
+                        "end":2.82675
+                        }
+                    ],
+                    "dst_segments":[
+                        {
+                        "start":0.0,
+                        "end":2.82675
+                        }
+                    ],
+                    "aug_function":"<lambda>",
+                    "output_path":null,
+                    "intensity":100.0
+                }
+            ],
+            "aug_function":"apply_lambda",
+            "audio_aug_kwargs":{
+                
+            },
+            "intensity":100.0
+            }
+        ],
```

Comment:
Please put the keys in alphabetical order (e.g. "audio_aug_kwargs" first etc), and insert spaces after the : to match the formatting of the rest of this metadata file :)

Reply:
Also "augment_audio" should actually go after "audio_swap", before "blend_videos", to be in alphabetical order.